com.ca.lsp.cobol/lsp-core-cobol-parser/src/main/java/com/ca/lsp/core/cobol/semantics/CobolVariableContext.java

```diff
@@ -94,6 +92,10 @@ public Variable get(String name) {
   public boolean parentContainsSpecificChild(String rootVariableName, String targetVariableName) {
     Variable rootVariable = get(rootVariableName);
 
+    if (rootVariable == null) {
```

Comment:
This if block can be removed, if rootVariable != null used, we have false return at the end.

Reply:
Ok, it seems reasonable to refactor this mthod